english/explore-outside/journey-to-flaming-tower/graham-cracker-cab/agree-to-pay/agree-to-pay.md

```diff
@@ -1,8 +1,26 @@
-Jumping in the cab you pay the greedy cab driver and he speeds you toward the burning tower.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.